CLAUDE.md

```
@@ -14,6 +14,20 @@ ArgoCD is bootstrapped here and deploys everything above via `nidavellir-apps.ya
 ```bash
 ./update-embedded-git.sh gke
 ```
+By default the script kubectl-port-forwards to `localhost:3000`. To push
+via the Gitea HTTPRoute instead (recommended on clusters where the
+ingress is wired up — and required when the local git credential
+manager intercepts localhost):
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)